rpcs3/Emu/Io/usio.cpp

```diff
@@ -221,10 +221,10 @@ void usb_device_usio::translate_input()
 
 void usb_device_usio::usio_write(u8 channel, u16 reg, const std::vector<u8>& data)
 {
-	auto write_memory = [&](std::vector<u8>& memory)
+	auto write_memory = [&](std::vector<u8>& memory, b8 exact_size = true)
```

Comment:
b8 is supposed to be used in structure inside PS3 memory instead of bool to ensure its single byte length, that's the only reason for its use and it shouldn't be used otherwise.

Reply:
> b8 is supposed to be used in structures residing in PS3 memory instead of bool to ensure its single byte length, that's the only reason for its use and it shouldn't be used otherwise.

I see. I've updated it.
Thank you.